controller/error_pages.rst

```diff
@@ -193,61 +180,95 @@ for a given status code as HTML or for a given status code and format.
      http://localhost/index.php/_error/{statusCode}
      http://localhost/index.php/_error/{statusCode}.{format}
 
-.. _custom-exception-controller:
-.. _replacing-the-default-exceptioncontroller:
+.. _overriding-non-html-error-output:
+
+Overriding Error output for non-HTML formats
+--------------------------------------------
 
-Overriding the Default ExceptionController
-------------------------------------------
+To override non-HTML error output, the Serializer component needs to be installed.
+
+.. code-block:: terminal
+
+    $ composer require serializer
+
+The Serializer component has a built-in ``FlattenException`` normalizer (``ProblemNormalizer``) and JSON/XML/CSV/YAML
+encoders by default. That means that if an exception were to be thrown in your application, Symfony can output it in
+a format supported by one of the encoders. If you want to change how the output is structured, all you have to do
+is create a new Normalizer that supports the ``FlattenException`` input.
```

Comment:
...end with „::“ instead of a „.“ this will also introduce the parser to parse the next block as code block php. Thanks